src/storages/inRedis/index.ts

```diff
@@ -40,8 +42,11 @@ export function InRedisStorage(options: InRedisStorageOptions = {}): IStorageAsy
       splits: new SplitsCacheInRedis(log, keys, redisClient),
       segments: new SegmentsCacheInRedis(log, keys, redisClient),
       impressions: new ImpressionsCacheInRedis(log, keys.buildImpressionsKey(), redisClient, metadata),
+      impressionCounts: impressionsMode !== DEBUG ? new ImpressionCountsCacheInMemory() : undefined,
```

Comment:
Emma, InRedisStorage needs its own `ImpressionCountsCacheInRedis` with a `track` method.  If you prefer, you can implement it in a different PR and place a `@TODO` comment here